Update searchCategory query examples and responses (#337)
AdobeDocs/commerce-services: src/pages/optimizer/merchandising-services/categories-storefront-implementation.md

## searchCategory query examples

The `searchCategory` query matches category **names** against a `searchTerm` and returns a paginated [`SearchCategoryResultPage`](https://developer.adobe.com/commerce/services/graphql-api/merchandising-api/index.html#definition-SearchCategoryResultPage).

> - If the `family` argument is omitted, it returns categories across all families. 

Author: ```suggestion
- If the `family` argument is omitted, it returns categories across all families.
```

> -  The `searchTerm` argument is case-insensitive and must be a minimum of three characters.

Reviewer: ```suggestion
- The `searchTerm` argument is case-insensitive and must be a minimum of three characters.
```

> - To limit search results, The `searchTerm` is case-insensitive and must be a minimum of three characters.

Author: ```suggestion
-  The `searchTerm` argument is case-insensitive and must be a minimum of three characters.
```

> Each `items` entry is a [`CategoryTreeView`](https://developer.adobe.com/commerce/services/graphql-api/merchandising-api/index.html#definition-CategoryTreeView), so you can reuse the same fields as in `categoryTree` responses (for example, `slug`, `name`, `description`, and `images`). 

Author: ```suggestion
Each `items` entry is a [`CategoryTreeView`](https://developer.adobe.com/commerce/services/graphql-api/merchandising-api/index.html#definition-CategoryTreeView), so you can reuse the same fields as in `categoryTree` responses (for example, `slug`, `name`, `description`, and `images`).
```

### Search categories by name

> Shoppers and internal tools often find categories by typing a fragment of the display name rather than browsing the tree. The following example calls `searchCategory` with `searchTerm: "Men"` and no `family`, then requests `totalCount`, a page of `items` with `slug`, `name`, and `level`, plus `pageInfo` for pagination. The response lists every category whose name matches across the catalog scope your API uses.

Reviewer: ```suggestion
Shoppers and internal tools often find categories by typing a fragment of the display name rather than browsing the tree. The following example calls `searchCategory` with `searchTerm: "Men"` and no `family`, then requests `totalCount`, a page of `items` with `slug`, `name`, and `level`,  plus `pageInfo` for pagination.. The response lists every category whose name matches across the catalog scope your API uses.
```

You removed pageInfo from your query. I'm not sure if you really should, given you link to it below. But then, I'm only looking at a diff, not the whole topic.
